korangar/src/world/animation/mod.rs

```diff
@@ -154,4 +182,41 @@ impl AnimationData {
             });
         }
     }
+
+    #[cfg(feature = "debug")]
+    pub fn render_debug(
+        &self,
+        instructions: &mut Vec<DebugRectangleInstruction>,
+        camera: &dyn Camera,
+        entity_position: Point3<f32>,
+        animation_state: &AnimationState,
+        head_direction: usize,
+        color_external: Color,
+        color_internal: Color,
+    ) {
+        const TILE_SIZE: f32 = 10.0;
```

Comment:
Do we need to repreat the constants here? Can't we move them to the top of the file and re-use them?

Reply:
I will put on top of the file